cfgov/unprocessed/css/molecules/social-media.less

```diff
@@ -0,0 +1,64 @@
+/* topdoc
+  name: Social Media
+  family: cfgov-molecules
+  patterns:
+    - name: Default example
+      markup: |
+        <div class="m-social-media">
+            <h5 class="m-social-media_heading">Share this</h5>
+            <ul class="list">
+                <li class="list_item">
+                    <a class="m-social-media_icon" href="#">
+                        <span class="cf-icon cf-icon-email-social-square"></span>
+                        <span class="u-visually-hidden">
+                            Share by email
+                        </span>
+                    </a>
+                </li>
+            </ul>
+        </div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .m-social-media
+            .m-social-media_heading
+            ul.list
+              li.list_item
+                .m-social-media_icon
+                  span.cf-icon
+                  span.u-visually-hidden
+  tags:
+    - cfgov-molecules
+*/
+
+.m-social-media {
+    &_heading {
+        display: inline-block;
+        padding-right: unit( @grid_gutter-width / @base-font-size-px, em );
+        vertical-align: middle;
+    }
+
+    .list {
+        display: inline-block;
+        padding: 0;
+        list-style: none;
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/fd3d39bd6066c1addf8890c2aeac14b4867c4773